cpp/src/arrow/compute/kernels/aggregate_basic.cc

```diff
@@ -482,8 +487,8 @@ void AddFirstOrLastAggKernel(ScalarAggregateFunction* func,
 // ----------------------------------------------------------------------
 // MinMax implementation
 
-Result<std::unique_ptr<KernelState>> MinMaxInit(KernelContext* ctx,
-                                                const KernelInitArgs& args) {
+Result<std::unique_ptr<KernelState>> MinMaxInitNative(KernelContext* ctx,
```

Comment:
We have `MeanImplDefault` and `MinMaxInitNative` two styles, lol

Reply:
Fixing.